sources/styles.css

```diff
@@ -147,3 +143,296 @@ div.button i.fa-arrow-down {
+  padding-right: 12px;
+  padding-bottom: 8px;
+  padding-left: 12px;
+  font-family: "Inter", sans-serif;
+  font-weight: 600;
+  font-style: normal;
+  font-size: 12px;
+  line-height: 16px;
+  letter-spacing: 3%;
+}
+
+ul.button-grid li.lang-button4 {
+  display: flex;
+  width: 51px;
+  height: 32px;
+  list-style: none;
+  box-sizing: border-box;
+  border: 1px solid #ebf0eb;
+  background-color: #ebf0eb;
+  border-radius: 4px;
+  padding-top: 8px;
+  padding-right: 12px;
+  padding-bottom: 8px;
+  padding-left: 12px;
+  font-family: "Inter", sans-serif;
+  font-weight: 600;
+  font-style: normal;
+  font-size: 12px;
+  line-height: 16px;
+  letter-spacing: 3%;
+}
+
+section.main-button {
+  display: flex;
+  align-content: center;
+  justify-content: center;
+  align-self: center;
+}
+
+button.projectview {
+  background-color: #36b37f;
+  color: #ffff;
+  margin-top: 29px;
+  margin-bottom: 24px;
+  width: 122px;
+  height: 48px;
+  border-radius: 4px;
+  font-family: "Inter", sans-serif;
+  font-weight: 500;
+  font-style: normal;
+  font-size: 17px;
+  line-height: 24px;
+  letter-spacing: 3%;
+}
+
+button.projectview:hover {
+  background-color: #3f9973;
+}
+
+button.projectview:active {
+  background-color: #008552;
+  box-shadow: 0 5px rgb(47, 46, 46);
+}
+
+/* about me */
+section.about {
+  display: grid;
+  grid-template-rows: auto auto auto auto;
+  box-sizing: border-box;
+  margin-top: 18px;
+  width: 100%;
+  justify-content: center;
+}
+
+div.aboutme-desc h2 {
+  display: flex;
+  justify-content: center;
+  width: 351px;
+  height: 48px;
+  font-family: "Inter", sans-serif;
+  font-weight: bold;
+  font-size: 40px;
+  line-height: 52px;
+  color: #172b4d;
+  margin-bottom: 24px;
+}
+
+div.aboutme-desc p {
+  display: flex;
+  text-align: center;
+  width: 351px;
+  height: 189px;
+  font-family: "Inter", sans-serif;
+  font-weight: 400;
+  font-style: normal;
+  font-size: 20px;
+  line-height: 32px;
+  color: #172b4d;
+}
+
+section.resume-btn {
+  display: flex;
+  justify-content: center;
+}
+
+section.resume-btn button {
+  background-color: #36b37f;
+  color: #ffff;
+  margin-top: 29px;
+  margin-bottom: 24px;
+  width: 122px;
+  height: 48px;
+  border-radius: 4px;
+  font-family: "Inter", sans-serif;
+  font-weight: 500;
+  font-style: normal;
+  font-size: 17px;
+  line-height: 24px;
+  letter-spacing: 3%;
+}
+
+section.about-container {
+  display: flex;
+  box-sizing: border-box;
+  border: 1px solid black;
+  background-color: #ebf0eb;
+  flex-direction: column;
+  align-items: center;
+  border-radius: 8px;
+  height: 352px;
+  width: 327px;
+  margin-top: 15px;
+  margin-left: 8px;
+  margin-right: 5px;
+  margin-bottom: 12px;
+}
+
+ul.container-project-skills {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  width: 100%;
+}
+
+ul.container-projects-skill {
+  display: flex;
+  flex-direction: row;
+  list-style-type: none;
+}
+
+.container-projects-skill li.secs {
```

Comment:
- [ ] Kindly add the correct spacing inside the button so it better matches the design
Yours:
![image](https://user-images.githubusercontent.com/73140461/148590774-8fd3e364-54a0-4b21-810e-b9da75c84ca9.png)
Figma:
![image](https://user-images.githubusercontent.com/73140461/148590786-330f5451-a2f8-4175-9d56-1d19af360c3d.png)